### DISCUSSION TITLE
Roadmap for the project

### DISCUSSION THREAD

**Discussion by aarcex3** [olxxi/ReqFlow#2] [Q&A] [answered]
Hi, first of all let me say that this is a very interesting project.
Are there any future plan for it?

**Comment by olxxi** [comment, marked as answer]
Hello,

Thank you very much for the feedback!

First of all, is stabilization and gathering user feedback on the missed/desired functionality since the project is in the alpha state. Talking about new functionality, I believe it would be great to leverage httpx functionality for async requests. Also, report refinement and additional integration with pytest.

If you have any ideas please feel free to share :)
